ts/Series/Flags/FlagsSeries.ts

```diff
@@ -526,25 +526,23 @@ class FlagsSeries extends ColumnSeries {
             );
 
             points.forEach(function (point): void {
-                const box = point.graphic && boxesMap[point.plotX as any];
-
-                if (box) {
-                    (point.graphic as any)[
-                        (point.graphic as any).isNew ? 'attr' : 'animate'
-                    ]({
-                        x: (box.pos as any) + (box.align as any) * box.size,
-                        anchorX: point.anchorX
-                    });
+                const plotX = point.plotX as number;
+                const box = point.graphic && boxesMap[plotX];
```

Comment:
Since we're referring in so many places to `point.graphic`, it is probably better to store it in a local variable in the first place.